app/views/layouts/peoplefinder/email.html.haml

```diff
@@ -0,0 +1,64 @@
+!!! Strict
```

Comment:
If we're going to use ERB for emails (a decision I endorse), can we do so everywhere?

This might help: https://github.com/danchoi/herbalizer